doc/scenario.md

```diff
@@ -439,3 +440,28 @@ Input facts (set by config):
 
 - [DEPRECATED] `cartridge_failover` - indicates if eventual failover should be enabled or disabled;
 - `cartridge_failover_params` - failover parameters.
+
+### cleanup
+
+Remove temporary files from `temporary_files` list. By default, `temporary_files` is an empty list. The role, depending
```

Comment:
It doesn't remove files from list.

`Removes temporary files specified in `temporary_files` list.`